### ISSUE TITLE
[WIIU] Content rotation causes rotation issues in RetroArch menus.

### ISSUE THREAD

**Issue by vaguerant** [libretro/RetroArch#11166]
## Description

In a previous (closed) issue I filed about graphical menu drivers being broken on RetroArch (#10476), I noted in a comment that using RetroArch rotation causes moderate-to-significant issues in the RetroArch GUI. When the initial issue described in 10476 was resolved, the issue was marked closed, but graphical menus remain broken due to the rotation problem. Quoting myself from that issue:

> The basic gist is that video rotation seems to affect the graphical menus when it should only be affecting the loaded content. e.g. In Ozone, the highlight bar indicating which menu item you currently have selected becomes rotated, while the rest of the menu is still upright:

> Video Rotation: Normal
> ![](https://user-images.githubusercontent.com/5259025/79832648-3a7d4f00-83ed-11ea-819c-cccf941e6780.jpg)

> Video Rotation: 90 deg
> ![](https://user-images.githubusercontent.com/5259025/79831800-77e0dd00-83eb-11ea-9c8a-1148828257ae.jpg)

> The effect on XMB is minimal, it seems to rotate the background gradient, which doesn't affect usability at all but is still unintended behavior, of course.

### Expected behavior

Content rotation settings should only affect content, not RetroArch menus.

### Actual behavior

Content rotation settings affect content *and* RetroArch menus.

### Steps to reproduce the bug

Enable any rotation setting other than "normal".

### Bisect Results

> The best I can do for the age of the menu rotation bug is "it's old". The earliest stable on the buildbot is 1.7.5 (826c77d) from October 3, 2018 and the rotation caused menu issues at least that far back.

### Version/Commit

- RetroArch: 2a9915f

### Environment information

- OS: Wii U

**Comment by paulwelka** [comment]
Any update on this issue? Changing to a Video Rotation other than “Normal” makes the Ozone menu almost unusable (on the Wii U platform) as described.

**Comment by vaguerant** [comment]
@paulwelka Can you still reproduce this issue? I fixed a different rotation issue on Wii U and was going to look around for what caused this bug, but I can't get it to happen any more.

**Comment by paulwelka** [comment]
Hi there. I noticed a rotation issue was fixed. Specifically, if I load and launch a vertical shooter game (such as 1941.zip with a core like MAME 2003+), I don't need to fiddle around with video options (the game renders in the correct, vertical orientation). In the past, that was broken. However, the menu issue remains with the Ozone GUI. Reproduce it as follows: 1) Load and launch a vertical shooter game like 1941.zip with a core like MAME 2003+. 2) Hit the Home button on the Wii U gamepad to display the "quick menu". The menu is readable from top to bottom, as it should be, but the "cursor" bar for navigating through the menu is rotated 90 degrees and moves left/right instead of up/down. This is happening in the Ozone GUI. I'm running the latest nightly build.

**Comment by vaguerant** [comment]
Yeah, this is a confusing one. I've been talking with @Ploggy, who still has the same issue, and we can't figure out why it's reproducible on their system but not mine. I *used* to have this issue (I filed the original report, after all), but at some point it seems to have stopped happening on my system.

![No problems here.](https://matrix-client.matrix.org/_matrix/media/r0/download/matrix.org/nFfimmjbbKqYItPMXQtNSktb)

No matter what we tried, Ploggy had the issue but I didn't:
- confirming Settings > Core > Allow Rotation is enabled
- running the same exact nightly build
- running the exact same game (we both tested `19xx`)
- running RetroArch at the same resolution (1080p)
- both having shaders disabled
- both launching from the Homebrew Launcher channel (not from the RA channel, etc.)
- running the same Salamander launcher (the main `retroarch.rpx`)
- using a fresh `retroarch.cfg`

Here's the things we know are definitely different on our systems:
- I use Haxchi, Ploggy uses coldboot Haxchi

And things that *might* be different on our systems:
- I use GaryOderNichts's [Controller Channel v2](https://github.com/wiiu-controller-mods/homebrew_launcher/releases/tag/channel_v2) Homebrew Launcher fork, Ploggy thinks they have the same mod

Inconveniently, I don't know when I stopped experiencing this bug. I gave up on playing vertical games around the time I first filed this issue (August 10, 2020) and only started messing with rotation again about a week ago, at which point I discovered the bug just doesn't happen any more on my system. I really can't think of anything else to explain why the bug stopped occurring on my end.

**Comment by alfonsico** [comment]
Same issue here
Retroarch version 1.15
Play in vertical mode show a rotate menu almost unusable 

any clue for fix it?
